quinn-proto/src/bloom_token_log.rs

```diff
@@ -0,0 +1,330 @@
+use std::{
```

Comment:
This is another commit with a lot of stuff going on. Suggest splitting it up in multiple commits (or PRs). For example, it seems that we could get most of the interfaces/traits in place before we actually slot in `BloomTokenLog`?

Reply:
Done